From d7b0a0ef152312b9242cbbe7405ee49310e7909e Mon Sep 17 00:00:00 2001
From: Matthias Maier <tamiko@43-1.org>
Date: Sun, 1 Mar 2020 11:50:14 -0600
Subject: [PATCH] CMake: Mark the libmsc library static

With CMake 3.16.4 at least we end up with a shared library and
subsequent linking fails.
---
 libmscgen/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmscgen/CMakeLists.txt b/libmscgen/CMakeLists.txt
index 079fcfc..fbf3f4f 100644
--- a/libmscgen/CMakeLists.txt
+++ b/libmscgen/CMakeLists.txt
@@ -16,7 +16,7 @@ foreach(lex_file ${LEX_FILES})
     FLEX_TARGET(${lex_file}        ${lex_file}.l        ${GENERATED_SRC}/${lex_file}.cpp        COMPILE_FLAGS "${LEX_FLAGS}")
 endforeach()
 
-add_library(mscgen
+add_library(mscgen STATIC
 gd.c
 gd_security.c
 gdfontt.c
-- 
2.24.1

